cmd/apps/openfaas_app.go

```diff
@@ -233,11 +233,25 @@ func mergeFlags(existingMap map[string]string, setOverrides []string) error {
 		if len(flag) != 2 {
 			return fmt.Errorf("incorrect format for custom flag `%s`", setOverride)
 		}
+
+		if strings.ContainsAny(flag[1], ",") {
+			flag[1] = handleCommaSeparatedValues(flag[1])
+		}
 		existingMap[flag[0]] = flag[1]
 	}
 	return nil
 }
 
+func handleCommaSeparatedValues(values string) string {
```

Comment:
Could you add a unit test for this method?

What about strings.Replace instead?

Reply:
@alexellis I updated it with `strings.ReplaceAll`, added unit test as well.